libraries/common-arm/memory.x

```diff
@@ -0,0 +1,7 @@
+MEMORY
+{
+  /* NOTE K = KiBi = 1024 bytes */
+  FLASH (rx) : ORIGIN = 0x00000000, LENGTH = 256K
```

Comment:
Now using the memory size for the ATSAME51J18 (the one we'll actually use on the rocket)

Reply:
Does the memory layout defined here in this 'common-arm' library get used by all boards that define this package as a dependency? Trying to figure out what this package is being used for.

Reply:
Yes, this file just needs to be present in the dependency tree. Also, my idea is that `common-arm` will eventually become the main crate, containing any non-board specific code. I've started to use it a bit more in the error manager PR.

Reply:
Ah, that makes more sense. Thanks!